libs/s25main/world/MapBase.cpp

```diff
@@ -182,3 +182,9 @@ MapPoint MapBase::MakeMapPoint(Position pt) const
 {
     return ::MakeMapPoint(pt, size_);
 }
+
+unsigned MapBase::numPointsRadius(unsigned radius, bool includePt) const {
```

Comment:
Ah I already had a formula for that :D

I'd suggest to make this a `constexpr` function in the header to allow for better optimizations and usage in template arguments (as you tried earlier). Simply replace the semicolon at https://github.com/Return-To-The-Roots/s25client/pull/1501/files#diff-61d8eac8fa8b6b4b316044fff0023aeb2eccf6858afb2e54ee4a6faa90558d1dR98 by the part between and including the braces and put `constexpr` in front of the return type. Done, no other changes required.